src/integration-test/java/com/mongodb/jdbc/integration/testharness/DataLoader.java

```diff
@@ -57,7 +60,20 @@ public class DataLoader {
                     + ":"
                     + System.getenv("ADF_TEST_LOCAL_PWD")
                     + "@localhost";
-    private static Yaml yaml = new Yaml(new Constructor(TestData.class));
+    private static Yaml yaml;
+
+    static {
+        TagInspector allowGlobalTags =
```

Comment:
Nice thanks for upgrading snakeyaml.  If I understand correctly, is this change to support the use of tags in the yaml such as `!!org.bson.BsonTimestamp`?

Reply:
ah, yes, you are correct that's what it's for.
I should have mentioned that. I also marked Snakeyaml as test dependency only so that it is not bundled and also not an issue to allow global tags.

I upgraded SnakeYaml and guava because they both have vulnerabilities reported.
I did that when listing dependencies and forgot to split the PR.